dotnet/sample/AutoGen.BasicSamples/Example11_Sequential_GroupChat_Example.cs

```diff
@@ -30,8 +31,15 @@ public static async Task<IAgent> CreateBingSearchAgentAsync()
         kernelBuilder.Plugins.AddFromObject(webSearchPlugin);
 
         var kernel = kernelBuilder.Build();
-        var kernelAgent = new SemanticKernelAgent(
-            kernel: kernel,
+
+        var agent = new ChatCompletionAgent()
+        {
+            Kernel = kernel,
+            Name = "ExampleAgent"
```

Comment:
You might need to add ExecutionSettings here so it can use bing search?

Reply:
Yeah good point, I overlooked that.